app/invitations/api.py

```diff
@@ -0,0 +1,102 @@
+import logging
+
+from fastapi import APIRouter, Depends
+from fastapi import status as http_status
+from starlette.responses import JSONResponse
+
+from app.core.auth_jwt_bearer import JWTBearer
+from app.invitations.model import (
+    CreateInvitationData,
+    InvitationResponse,
+)
+
+logger = logging.getLogger("api.invitations")
+router = APIRouter()
+
+
+@router.post(
+    path="",
+    status_code=http_status.HTTP_201_CREATED,
+    response_model=InvitationResponse,
+    dependencies=[Depends(JWTBearer())]
+)
+async def create_invitations(
+        data: CreateInvitationData,
+):
+    """
+    Require User Authentication Token
+
+    Send an invitation email to a new user within an organisation
+    """
+    return JSONResponse(status_code=201,
+                        content={"message": "Invitation has been sent"})
```

Comment:
I'd replace these ones with `raises NotImplementedError` or similar as it can be misleading like that